conf/modules.config

```diff
@@ -626,6 +626,36 @@ process {
         ]
     }
 
+    withName: ARGNORM_ABRICATE {
+        publishDir = [
+            path: {"${params.outdir}/arg/hamronization/abricate/argnorm/"},
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.prefix = { "${meta.id}.normalized.tsv" }
+        ext.args = "--hamronized"
+    }
+
+    withName: ARGNORM_AMRFINDERPLUS {
+        publishDir = [
+            path: {"${params.outdir}/arg/hamronization/amrfinderplus/argnorm/"},
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.prefix = { "${meta.id}.normalized.tsv" }
+        ext.args = "--hamronized"
```

Comment:
Oh, so I actually meant that if an argnorm supported database isn't used we don't even have to run argnorm in that case.  `hamronization` would only change the column from which argNorm accesses the ARGs, if an unsupported database is used with `amrfinderplus`, argNorm won't be able to normalize even if it isn't processed using `hamronization`.

Reply:
If ARGNORM doesn't work with specific databases, we should add some code that if the database selected by the user is not compatible, the module should not execute. We can either do this here with a `ext.when = <condition>`, or an if/else around the module in the code, or filter out the hamronization files in the channel that goes into the ARGNORM module

Reply:
I think this would only apply to abricate though in this case, different named amrfinderplus  databases is not supported (you can supply a path, but I don't think it uses distinct names), and deeparg has a single database.

Reply:
Ah, I forgot about abricate! Thanks for pointing that out, I'll add the conditions for abricate then